test/e2e/tests/auth.setup.ts

```diff
@@ -0,0 +1,53 @@
+import { test as setup } from '@playwright/test';
+import path from 'path';
+
+import { navigateToAppointmentAndSignIn, setDefaultUserSettingsLocalStore } from '../utils/utils';
+
+import {
+    APPT_DASHBOARD_HOME_PAGE,
+    APPT_MAIN_SETTINGS_PAGE,
+    TIMEOUT_2_SECONDS,
+} from "../const/constants";
+
+const fs = require('fs');
+const directoryPath = path.join(__dirname, '../test-results/.auth');
+
+fs.mkdir(directoryPath, (err: any) => {
+  if (err) {
+    console.error('error creating auth storage directory:', err);
```

Comment:
A nit: You can return after this line and get rid of the else to reduce indentations.